public/components/page-dashboard/select-contribution-month/select-contribution-month.js

```diff
@@ -5,64 +5,87 @@ import view from './select-contribution-month.stache';
 import ContributionMonth from 'bitcentive/models/contribution-month/';
 
 export const ViewModel = DefineMap.extend({
-  selectedContributionMonthId: {
```

Comment:
This whole file was re-written to take advantage of the new `value({resolve})` property behavior.